submissions/karelskiy/tiny-JS-world/index.js

```diff
@@ -0,0 +1,39 @@
+const population = [
+    {
+        species: 'dog',
+        name: 'Toby',
+        gender: 'male',
+        legs: 4,
+        hands: 0,
+        saying: 'woof-woof!'
+    },
+    {
+        species: 'cat',
+        name: 'Mrs. Velasquez',
+        gender: 'female',
+        legs: 4,
+        hands: 0,
+        saying: 'meow-meow!'
+    },
+    {
+        species: 'human',
+        name: 'Enrico',
+        gender: 'male',
+        legs: 2,
+        hands: 2,
+        saying: 'Holla'
+    },
+    {
+        species: 'human',
+        name: 'Angel',
+        gender: 'female',
+        legs: 2,
+        hands: 2,
+        saying: 'Ehi, buongiorno'
+    }
+    
+];
+
+population.forEach(el => {
+    print(`${el['species']}; ${el['name']}; ${el['gender']}; ${el['legs']}; ${el['hands']}; ${el['saying']}`);
```

Comment:
There is a pattern here.
How about mapping an ordered list of properties into values from `el` object and then joining the result with a delimiter?
This would make code [DRYer](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), better readable and scalable (e.g. when we want to add more properties).

Reply:
Thanks, will be done.